.github/workflows/commit-message-check.yaml

```diff
@@ -18,24 +18,26 @@ jobs:
     name: Commit Message Check
     steps:
     - name: Get PR Commits
+      if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
```

Comment:
Not a github workflow expert speaking...but It seems you can move the conditional `if` up at the jobs' definition level so you skip it entirely (instead of skip step-by-step). Example here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
No, unfortunately you cannot.  That assumption was in the first logic added and I had to fix it to get it to work as part of 3fe6695b0a7f0f1a132efa9b2ebd11f5e059f270.
```
commit 3fe6695b0a7f0f1a132efa9b2ebd11f5e059f270
Refs: {fidencio/wip/workflows-fix-skip-ci}, 2.2.0-alpha1-111-g3fe6695b0
Author:     Fabiano Fidêncio <fidencio@redhat.com>
AuthorDate: Fri Jul 30 12:14:45 2021 +0200
Commit:     Fabiano Fidêncio <fidencio@redhat.com>
CommitDate: Fri Jul 30 13:51:56 2021 +0200

    static-checks: Check for the `force-skip-ci` label on each step

    This is not the most beautiful solution, but when do the check on every
    single step we ensure the test at least started, and consequently will
    succeed.

    Without this the tests wouldn't even start, making any PR using the
    `force-skip-ci` label not mergeable.

    Fixes: #2362

    Signed-off-by: Fabiano Fidêncio <fidencio@redhat.com>
```

Reply:
I'm really surprised.... good to know :) Ignore my comment.